docs/Upgrade.rst

```diff
@@ -51,6 +51,28 @@ In this section, we will present all the breaking changes you should
 know before starting the upgrade to 6.0 version.
 
 
+**Architecture changes**
```

Comment:
Some of these changes are not breaking changes since they don't require actions from customers to upgrade. These changes should be moved in the release notes IMO.

Reply:
done